<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomationWidget</class>
 <widget class="QWidget" name="AutomationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Batch Processing Tasks</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="2" column="1">
       <widget class="QPushButton" name="importTasksButton">
        <property name="text">
         <string>Import New Tasks</string>
        </property>
        <property name="icon">
         <iconset resource="../UI_Icons.qrc">
          <normaloff>:/Resources/UISymbols/UI_Add_Task.svg</normaloff>:/Resources/UISymbols/UI_Add_Task.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="idleTasks"/>
      </item>
      <item row="1" column="4">
       <widget class="QListWidget" name="queuedTasks"/>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="queueAllButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../UI_Icons.qrc">
            <normaloff>:/Resources/UISymbols/UI_Move_Right.svg</normaloff>:/Resources/UISymbols/UI_Move_Right.svg</iconset>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="queueSelectedButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../UI_Icons.qrc">
            <normaloff>:/Resources/UISymbols/UI_Move_All_Right.svg</normaloff>:/Resources/UISymbols/UI_Move_All_Right.svg</iconset>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="unqueueAllButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../UI_Icons.qrc">
            <normaloff>:/Resources/UISymbols/UI_Move_Left.svg</normaloff>:/Resources/UISymbols/UI_Move_Left.svg</iconset>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="unqueueSelectedButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../UI_Icons.qrc">
            <normaloff>:/Resources/UISymbols/UI_Move_All_Left.svg</normaloff>:/Resources/UISymbols/UI_Move_All_Left.svg</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../UI_Icons.qrc">
            <normaloff>:/Resources/UISymbols/UI_Start_Icon.svg</normaloff>:/Resources/UISymbols/UI_Start_Icon.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../UI_Icons.qrc">
            <normaloff>:/Resources/UISymbols/UI_Pause_Icon.svg</normaloff>:/Resources/UISymbols/UI_Pause_Icon.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../UI_Icons.qrc">
            <normaloff>:/Resources/UISymbols/UI_Delete_Alt_Icon.svg</normaloff>:/Resources/UISymbols/UI_Delete_Alt_Icon.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="4">
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../UI_Icons.qrc"/>
 </resources>
 <connections/>
</ui>
